spring-integration-http/src/main/java/org/springframework/integration/http/inbound/HttpRequestHandlingEndpointSupport.java

```diff
@@ -148,8 +148,6 @@
 
 	private volatile Map<String, Expression> headerExpressions;
 
-	private volatile boolean shuttingDown;
```

Comment:
Note. I didn't reinstate this variable, because the `beforeShutdown()` from the `AmqpInboundChannelAdapter` fits better here anyway.